Tests/test_imagefont.py

```diff
@@ -326,23 +299,17 @@ def test_rotated_transposed_font(font, orientation):
 
     # Original font
     draw.font = font
-    with pytest.warns(DeprecationWarning) as log:
-        box_size_a = draw.textsize(word)
-        assert box_size_a == font.getsize(word)
-    assert len(log) == 2
     bbox_a = draw.textbbox((10, 10), word)
 
     # Rotated font
     draw.font = transposed_font
-    with pytest.warns(DeprecationWarning) as log:
-        box_size_b = draw.textsize(word)
-        assert box_size_b == transposed_font.getsize(word)
-    assert len(log) == 2
     bbox_b = draw.textbbox((20, 20), word)
 
     # Check (w,h) of box a is (h,w) of box b
-    assert box_size_a[0] == box_size_b[1]
-    assert box_size_a[1] == box_size_b[0]
+    assert (bbox_a[2] - bbox_a[0], bbox_a[3] - bbox_a[1]) == (
+        bbox_b[3] - bbox_b[1],
+        bbox_b[2] - bbox_b[0],
+    )
```

Comment:
Let's add in a magic comma to make this a bit more readable:
```suggestion
    assert (
        bbox_a[2] - bbox_a[0],
        bbox_a[3] - bbox_a[1],
    ) == (
        bbox_b[3] - bbox_b[1],
        bbox_b[2] - bbox_b[0],
    )
```